src/backend/joanie/core/admin.py

```diff
@@ -84,3 +84,15 @@ class AddressAdmin(admin.ModelAdmin):
     """Admin class for the Address model"""
 
     list_display = ("name", "address", "postcode", "city", "country", "owner", "main")
+
+
+@admin.register(models.CreditCard)
+class CreditCardAdmin(admin.ModelAdmin):
+    """Admin class for the credit card model"""
+
+    list_display = ("owner", "name", "expiration_date", "numbers", "main")
```

Comment:
some of these fields (owner? expiration_date?) should probably be readonly? If there is a problem with a credit card we will want to inspect, maybe delete it but not modify it as it was returned way PayBox on user secret input.